--- opensmtpd-201310231634p1/mk/makemap/Makefile.am~	2013-10-23 16:41:36.000000000 +0300
+++ opensmtpd-201310231634p1/mk/makemap/Makefile.am	2013-11-03 01:28:42.542110133 +0200
@@ -43,6 +43,12 @@
 CFLAGS+=		-D_GNU_SOURCE
 CPPFLAGS=		-I$(srcdir) @CPPFLAGS@ $(PATHS) @DEFS@
 
+PATHS=			-DSMTPD_CONFDIR=\"$(sysconfdir)\"			\
+			-DPATH_SMTPCTL=\"$(sbindir)/smtpctl\"			\
+			-DPATH_MAILLOCAL=\"$(pkglibexecdir)/mail.local\"	\
+			-DPATH_FILTERS=\"$(pkglibexecdir)\"			\
+			-DPATH_TABLES=\"$(pkglibexecdir)\"
+
 MANPAGES=		makemap.8.out newaliases.8.out
 MANPAGES_IN=		$(smtpd_srcdir)/makemap.8
 MANPAGES_IN+=		$(smtpd_srcdir)/newaliases.8
